app/src/socket-io.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+import { PayloadAction } from '@reduxjs/toolkit';
+import { io, Socket } from 'socket.io-client';
+import { AppState } from './store';
+
+export const socketIOUrl = 'http://127.0.0.1:4000/gateway';
```

Comment:
Should this be coming from ENV?

Reply:
Yeah totally, I should have left a comment. I've got a few more tasks in the epic to implement web sockets, and I'll slap it in `.env` in one of those.